microservices/rosettafold/microservice/api_models.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List
+
+import pydantic
+from pydantic import Field
+
+from microservice.mixins import BaseModelMixin
+
+
+@pydantic.dataclasses.dataclass
+class RunRosettaFoldRequest(BaseModelMixin):
+    fasta_content: str | None
+    a3m_content: str | None
+
+
```

Comment:
We also need:

```suggestion
@dataclasses.dataclass
class IsJobRunningResponse:
    is_running: bool
```

for similar purposes as from the comment above